src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPReport.kt

```diff
@@ -0,0 +1,110 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.helper.EntityOpen
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
```

Comment:
I've currently added minimal data for a CSIP report and associated plan.  Additional fields and other classes still to be added (separately, to avoid BIG PR)